controllers/baremetalset_controller.go

```diff
@@ -195,11 +194,12 @@ func (r *BaremetalSetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 
 	if op != controllerutil.OperationResultNone {
 		r.Log.Info(fmt.Sprintf("IPSet for %s successfully reconciled - operation: %s", instance.Name, string(op)))
+		return ctrl.Result{RequeueAfter: time.Second * 10}, nil
```

Comment:
We were missing these before. Important to have them in case it takes longer for the overcloudipsets to generate.